### PR TITLE
Added Meeting Room FAQ using uswds Accordion. Created shared folder f…

### PR DESCRIPTION
**Pull request by sterlingmuller**
Added MeetingRoom FAQ
Created share folder with constants file for FAQ information

*There is an error with the plus and minus icons from usdws not displaying. This issue was impacting other tickets and I believe Mark is looking into it 

[](https://github.com/APL-Innovation-Lab/meeting-room-prototype/issues/20)

Screenshot 2024-08-19 at 4 58 28 PM

### CODE REVIEW

File: app/components/MeetingRoomFAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            title: `${index + 1}. ${entry.question}`,
+            content: entry.answer,
+            expanded: false,
+            id: index.toString(),
+            headingLevel: "h4",
+        }),
+    );
+
+    return (
+        <div>
```
Comment (markmals) [inline app/components/MeetingRoomFAQ.tsx:17]: Is an element necessary here? And if so, should it be a `div` or something else?

File: app/shared/constants.tsx
Comment (markmals) [inline app/shared/constants.tsx:1]: I'm conflicted about mixing data and views like this. It's probably fine for the prototype, but I think I'd like to figure out a better pattern once we're building out the production site. Interested to hear opinions on this from other members of the team.

**Review by markmals — COMMENTED** [review]
Just one comment and one question. Looks good overall! I think this is a really elegant solution for this story.

File: app/shared/constants.tsx
Comment (julielaursen) [reply app/shared/constants.tsx:1]: +1

File: app/routes/_index.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { Card, CardGroup, CardHeader } from "@trussworks/react-uswds";
+import MeetingRoomFAQ from "~/components/MeetingRoomFAQ";
 import { mergeMeta } from "~/lib/merge-meta";
 
 export const meta = mergeMeta(({ parentTitle }) => [{ title: `Meeting Spaces • ${parentTitle}` }]);
 
 export default function Index() {
     return (
+        <>
```
Comment (julielaursen) [inline app/routes/_index.tsx:9]: Is <> necessary here

File: app/shared/constants.tsx
Comment (sterlingmuller) [reply app/shared/constants.tsx:1]: Word! Yeah, it was just a little tricky to implement some formatting when mapping over the QA. I can make this a string and write a helper function to format

File: app/routes/_index.tsx (hunk 2 above)
Comment (sterlingmuller) [reply app/routes/_index.tsx:9]: Yeah, a JSX expression must return a single parent element (we can't have both MeetRoomFAQ and CardGroup be the top level element). That being said, the empty fragment is more or less a placeholder, I'm sure it'll become a div with styling once more components get added to the screen.

File: app/components/MeetingRoomFAQ.tsx (hunk 1 above)
Comment (sterlingmuller) [reply app/components/MeetingRoomFAQ.tsx:17]: Ah, yeah this can be an empty fragment. Thanks!

File: app/shared/constants.tsx
Comment (markmals) [reply app/shared/constants.tsx:1]: I wouldn't worry about it for now, but it's something to keep in mind for the future

**Review by markmals — APPROVED** [review]
LGTM

**Pull request merged**